index.html

```diff
@@ -9,42 +9,61 @@
   </head>
 
   <body>
-    <nav class="nav">
-      <a href="/" class="nav__link" data-link>List</a>
-      <a href="/" class="nav__link" data-link>Add New</a>
-      <a href="/" class="nav__link" data-link>Contact</a>
-    </nav>
-    <div class="books" id="books">
-      <h1>Awesome Books</h1>
-      <table>
-        <tr>
-          <th>Title</th>
-          <th>Author</th>
-          <th></th>
-        </tr>
-        <tbody id="lists"></tbody>
-      </table>
+    <div class="navigation">
+      <h1 class="logo"><a href="">Awesome Books</a></h1>
+      <nav class="nav">
+        <a href="#" class="nav__link list" >List</a>
+        <a href="#" class="nav__link addbooks" >Add New</a>
+        <a href="#" class="nav__link contacts" >Contact</a>
```

Comment:
- [ ] OPTIONAL
We want to enhance the user experience of this application, and therefore kindly ensure that an **active menu link is highlighted**(color change or change in opacity). In other words, the current active page should have the corresponding menu item highlighted, as can be seen in the provided [wireframe design](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md). Kindly ensure to improve upon this aspect in your design, as we really want our designs to as much as possible mimic the provided wireframe template.

**Your nav menu:**
![image](https://user-images.githubusercontent.com/12745474/183249912-fbe9df90-b0e2-4539-a01c-ac345fb2d808.png)


**The wireframe nav menu:**
![image](https://user-images.githubusercontent.com/12745474/138342596-c4a68582-914a-4c5a-b255-8db0325cbaca.png)